source/reference/json/search-nodes-config-file.txt

```diff
@@ -0,0 +1,92 @@
+.. _atlas-cli-search-nodes-config-file:
+
+===============================
+Search Nodes Configuration File
+===============================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+You can use a configuration file to specify the required 
+settings for creating search nodes on a {+cluster+} 
+by using the {+atlas-cli+}. The {+atlas-cli+} accepts 
+``.json`` configuration files. 
+
+.. _search-nodes-settings:
+
+Search Nodes Settings
+---------------------
+
+You can specify the following settings in the search nodes
+configuration file. For a full list of settings and 
+descriptions, refer to the :oas-atlas-op:`API specification 
+</createAtlasSearchDeployment>`.
+
+.. list-table:: 
+   :header-rows: 1 
+   :widths: 20 10 70 
+
+   * - Field 
+     - Type 
+     - Description 
+
+   * - ``specs``
+     - Array of objects
+     - Settings to configure search nodes for your cluster.
+
+   * - ``specs.instanceSize``
+     - string
+     - Hardware specification for the search node instance sizes.
```

Comment:
I think this setting warrants adding a link to the search nodes information where NVME is discussed. Do we have such a place in the docs? I would link to it.